service/history/execution/workflow.go

```diff
@@ -182,7 +182,10 @@ func (r *workflowImpl) SuppressBy(
 		return TransactionPolicyPassive, nil
 	}
 
-	lastWriteCluster := r.clusterMetadata.ClusterNameForFailoverVersion(lastWriteVersion)
+	lastWriteCluster, err := r.clusterMetadata.ClusterNameForFailoverVersion(lastWriteVersion)
+	if err != nil {
+		return TransactionPolicyActive, err
```

Comment:
Without knowing this implementation well it's hard to know exactly what the correct behaviour is, but I guess this is in line with the other error handling